### DISCUSSION TITLE
How to use segments via Home Assistant

### DISCUSSION THREAD

**Discussion by ironicbadger** [Hypfer/Valetudo#760] [Q&A / Support] [answered]
Zoned cleanup has been working fine for a while but with the new segments features in the 2021.03.0 release it feels like they both do kind of the same thing.

Is it possible to dispatch the robovac to cleanup a specific segment via Home Assistant or MQTT or ...?

**Comment by Hypfer** [comment, marked as answer]
https://valetudo.cloud/pages/integrations/home-assistant-integration.html

**Reply by ironicbadger** [reply]
Thanks - how do you get the co-ordinates for segments?

**Reply by Hypfer** [reply]
If you zoom in on a segment marker, it will display both it's segment id as well as the segments' area in m².

**Reply by ironicbadger** [reply]
That is great!! I'd be happy to submit a PR to the documentation unless it's just easier for you to add that little piece of info.

**Reply by Hypfer** [reply]
Yes please!

**Reply by ironicbadger** [reply]
https://github.com/Hypfer/Valetudo/pull/761
